src/Components/Register/Register.module.scss

```diff
@@ -39,6 +39,11 @@
       margin: 1rem 0;
     }
 
+    .registerinputError {
+      margin: 1rem 0;
```

Comment:
Please sort this styles in alpha order ascending.

Reply:
Done.